docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.3'
+
+services:
+  web:
+    build: .
+    command: >
+      sh -c "python manage.py collectstatic --noinput && 
+             python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./movienager:/usr/src/movienager/
+    ports:
+      - 8000:8000
+    env_file: .env-dev
```

Comment:
Przydaje się też określenie nazwy obrazu - klucz `image`. Potem jak buduje się wielokrotnie to czasem przydaje się cofnąć do poprzedniej wersji obrazu i nazwa może być pomocna. Tak samo jak się wrzuca obrazy na dockerhub albo własny rejestr obrazów to już zdefiniowanie nazwy jest konieczne